### DISCUSSION TITLE
problem value objects quest mission

### DISCUSSION THREAD

**Discussion by hidre01** [BG-Software-LLC/SuperiorSkyblock2#1321] [Denied Feature Requests] [answered]
good evening i come glass you because i have a problem with another quest it is about the value of the quette it does not want to display i made a copy paste of the quest on the site to take the example what is the reason for that the quest does not display the good value please

`# The mission file to use
mission-file: ItemsMissions

auto-reward: true

reset-after-finish: false

rewards:
  commands:
    - 'eco give %player% 1000'
    - 'is admin msg %player% &e&lCollector | &7Successfully finished the mission Collector I!'
    - 'is admin msg %player% &e&lCollector | &7Now that you collected the nether star, lets look for the next item.'
    - 'is admin msg %player% &e&lCollector | &7&oFor more information about the next mission, checkout /is missions'

required-items:
  '1':
    types:
      - 'STONE'
    amount: 64

icons:
  not-completed:
    type: PAPER
    name: '&aCollectioneur I'
    lore:
      - '&7Collect x64 Stone.'
      - ''
      - '&6Required Items:'
      - '&8 - &7x64 Stone'
      - ''
      - '&6Rewards:'
      - '&8 - &7$1,000'
      - ''
      - '&6Stone Collected: &7{value_stone}/64'
      - '&6Progress: &7{0}%'
      - '&c&l ✘ &7Not Completed'
  can-complete:
    type: PAPER
    name: '&aCollectioneur I'
    lore:
      - '&7Collect x64 Stone.'
      - ''
      - '&6Required Items:'
      - '&8 - &7x64 Stone'
      - ''
      - '&6Rewards:'
      - '&8 - &7$1,000'
      - ''
      - '&6Stone Collected: &764/64'
      - '&6Progress: &7100%'
      - '&a&l ✔ &7Click to redeem your reward.'
    enchants:
      DURABILITY: 1
    flags:
      - HIDE_ENCHANTS
  completed:
    type: MAP
    name: '&aCollectioneur I'
    lore:
      - '&7Collect x64 Stone.'
      - ''
      - '&6Rewards:'
      - '&8 - &7$1,000'
      - ''
      - '&6Stone Collected: &764/64'
      - '&6Progress: &7100%'
      - '&a&l ✔ &7Already Claimed.`

<img width="237" alt="problem_value" src="https://user-images.githubusercontent.com/81771759/184778946-8a0dd1ba-73ea-4882-8215-a7f89f88805e.png">

**Comment by OmerBenGera** [comment, marked as answer]
Open an issue regarding this
